db.json

```diff
@@ -147,6 +147,20 @@
         }
       ],
       "standards": [
+        {
+          "name": "Cookie Store API",
+          "href": "https://wicg.github.io/cookie-store/",
+          "description": "An asynchronous Javascript cookies API for documents and service workers.",
+          "authors": [
+            {
+              "name": "Dylan Cutler",
+              "email": "dylancutler@google.com"
+            }
+          ],
+          "reference": "COOKIESTOREAPI",
+          "review_draft_schedule": [],
+          "twitter": "cookiestoreapi"
```

Comment:
Have you registered this account, or should we?

Reply:
I have not registered one yet, if WHATWG could register one that would be great, not sure how that is normally done.

Reply:
Alright, I registered it.

Our usage of Twitter is not really working these days; we used to have automatic bots that tweeted all substantial commits. Probably we should give up on that, but some of our tooling might break if we just leave this blank, so it was easiest just to register for now.